### PR TITLE
Add works & about myself section

### PR DESCRIPTION
**Pull request by AlexHamn**
Add and style works & about myself section. 
*Designed for mobile view.*

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -50,6 +48,87 @@ <h2>Software Developer</h2>
...
+          <img src="media/icons/peace.svg" alt="peace">
+          <img src="media/icons/twitter.svg" alt="twitter">
+          <img src="media/icons/m.svg" alt="m">
+        </div>
+        <div>
+          <h4>↓ Work</h4>
+        </div>
+        <div>
+          <h2>MY RECENT <br> WORK</h2>
+        </div>
```
Comment (Georjane) [inline index.html:66]: - [ ] I think this heading of MY RECENT WORK, should be in the next section of your projects. It's a heading that describes what that section is all about.
- [ ] Also there is an overlap of this heading with the first image of your projects. The heading is completely hidden behind the image. I had to increase the screen width to notice that it was hidden behind. See screenshot below:
image

File: index.html [hunk 2]
```diff
@@ -50,6 +48,87 @@ <h2>Software Developer</h2>
...
+            when an unknown printer took a standard dummy text.</p>
+          <div class="container languages">
+            <p>css</p>
+            <p>html</p>
+            <p>bootstrap</p>
+            <p>Ruby</p>
+          </div>
+          <button>
+          See Project
+          </button>
```
Comment (Georjane) [inline index.html:113]: - [ ] The button ("See project") is a button tag which you did correctly, should follow the button styling: Active, Hover, and Pressed (see design below)
image
Please follow this design for all your buttons as it makes your User Experience even more better.

File: index.html [hunk 3]
```diff
@@ -50,6 +48,87 @@ <h2>Software Developer</h2>
...
+          <h3>Multi-Post Stories</h3>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. 
+            has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a standard dummy text.</p>
+          <div class="container languages">
+            <p>css</p>
+            <p>html</p>
+            <p>bootstrap</p>
+            <p>Ruby</p>
+          </div>
```
Comment (Georjane) [inline index.html]: - [ ] One of the requirements for this project is that. the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should all be a list `ul` and each of them a `li`. This helps you to write semantic code which is a good practice. Please correct this for all other project sections in your portfolio

File: index.html [hunk 4]
```diff
@@ -50,6 +48,87 @@ <h2>Software Developer</h2>
...
         </ul>
       </section>
+      <section class="grid work">
+        <div class="container handles">
+          <img src="media/icons/github.svg" alt="github">
+          <img src="media/icons/linkedin.svg" alt="linkedin">
+          <img src="media/icons/peace.svg" alt="peace">
+          <img src="media/icons/twitter.svg" alt="twitter">
+          <img src="media/icons/m.svg" alt="m">
+        </div>
```
Comment (Georjane) [inline index.html]: - [ ] In order to make use of semantic tags, this list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)

File: styles.css [hunk 5]
```diff
@@ -23,17 +24,30 @@
...
+  grid-auto-rows: minmax(100px, auto);
+}
+
+/* .grid > * {
+  background: burlywood;
+}
+
+.grid > *:nth-child(odd) {
+  background: tomato;
+} */
```
Comment (Georjane) [inline styles.css]: - [ ] Please can you remove any commented out code from your project, It helps to keep your code clean. And besides you can always use git versioning to see any previous code you wrote.

**Review by Georjane — CHANGES_REQUESTED** [review]
### Hello, @AlexHamn, ✋🏽 

Good job so far! You have a very professional Readme file, and you have a good mastery of git-flow. Your linters are all passing, I applaud that too. ✔️ 
However, we have a few more changes to make in order to meet all requirements for this project but you are almost there!

## Required Changes ♻️

- [ ] In your Readme file, in the section where you have Setup, you should use the clone command with the actual URL of your project, like this 
```
git clone https://github.com/AlexHamn/portfolio.git
cd portfolio
```

- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- You can add a live link to your Readme file, that way anyone viewing your repo can easily see your project directly without having to clone it locally.

Cheers and Happy coding! 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 🤝 

_Code Reviewer
**[Jane](https://witahgeorjane.tech/)**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: styles.css (hunk 5 above)
Comment (rloterh) [reply styles.css]: Well resolved

File: index.html (hunk 2 above)
Comment (rloterh) [reply index.html:113]: Well resolved

File: index.html (hunk 3 above)
Comment (rloterh) [reply index.html]: Well resolved

File: index.html (hunk 4 above)
Comment (rloterh) [reply index.html]: Resolved

File: index.html (hunk 1 above)
Comment (rloterh) [reply index.html:66]: Well resolved

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Alex, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it  👍.
Congratulations! 🥂


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**